lua/nordic/groups/integrations.lua

```diff
@@ -97,39 +97,83 @@ function M.get_groups()
     G.MiniStatuslineFileinfo = { bg = LC.normal.b.bg, fg = LC.normal.b.fg }
     G.MiniStatuslineInactive = { bg = LC.inactive.a.bg, fg = LC.inactive.a.fg, bold = LC.inactive.a.gui == 'bold' }
 
+    -- Tree Groups
+    G.TreeNormal = { fg = C.fg, bg = C.bg }
+    G.TreeNormalNC = { fg = C.fg, bg = C.bg }
+    G.TreeRootFolder = { fg = C.gray4 }
+    G.TreeRootName = { fg = C.white2 }
+    G.TreeFileIcon = { fg = C.blue2 }
+    G.TreeFileNameOpened = { fg = C.fg }
+    G.TreeSpecialFile = { fg = C.magenta.bright }
+    G.TreeGitConflict = { fg = C.magenta.bright }
+    G.TreeGitModified = { fg = C.git.change }
+    G.TreeGitDirty = { fg = C.gray4 }
+    G.TreeGitAdded = { fg = C.git.add }
+    G.TreeGitNew = { fg = C.gray4 }
+    G.TreeGitDeleted = { fg = C.gray4 }
+    G.TreeGitStaged = { fg = C.gray4 }
+    G.TreeGitUntracked = { fg = C.orange.base }
+    G.TreeTitleBar = { fg = C.gray2, bg = C.white3 }
+    G.TreeFloatBorder = { fg = C.white3, bg = C.gray0 }
```

Comment:
These seem out of the norm for this theme. I can't really think of what color should be used, but I will mull it over.

Reply:
ok, if you got something let me know

Reply:
They should probably use `bg_float` (will check now what it is called) since float colors depends on the config

Reply:
Take a look here https://github.com/AlexvZyl/nordic.nvim/blob/main/lua/nordic/colors/init.lua

Reply:
it's bg_float, fg_float for TreeTitleBar and fg_float_border, bg_float_border for TreeFloatBorder as I can see

Reply:
> Take a look here https://github.com/AlexvZyl/nordic.nvim/blob/main/lua/nordic/colors/init.lua

I am not sure about that they are more like a title: 
![image](https://github.com/user-attachments/assets/f89eed5d-863d-4058-ad0d-4b1a2b702111)

Reply:
The most obvious titles we have are for telescope but I am not sure those work here either:
![image](https://github.com/user-attachments/assets/4f6f632b-8528-4c32-94d3-d6cdbca9b16a)

Reply:
i'd say the blue one looks good. I can try them and send screenshots and tell me which one is better

Reply:
can you just give me the colors for fg and bg?

Reply:
Not the right text color but just background examples:
![image](https://github.com/user-attachments/assets/58d322a9-6513-4a2a-8ff4-8b8bdfde59c4)
![image](https://github.com/user-attachments/assets/dc3ac607-39ae-4d73-85d7-46150102e0b1)

Reply:
> can you just give me the colors for fg and bg?

This line: https://github.com/AlexvZyl/nordic.nvim/blob/1ee4044077059b34eacd18d93613495a602c131f/lua/nordic/groups/integrations.lua#L259

orange.base and blue2

Reply:
ok, I would say that the blue one fits better. if y'all think otherwise tell me

Reply:
All the other (maybe) non reserved non grayscale ones:
![image](https://github.com/user-attachments/assets/066b7dee-f84d-449e-887c-69325b297afa)
![image](https://github.com/user-attachments/assets/f4421498-1a44-4408-9a1b-c40ae6dab82e)
![image](https://github.com/user-attachments/assets/9191da6b-4f77-4aeb-86ff-16c4170d631b)
![image](https://github.com/user-attachments/assets/b38d9c21-c590-4ed4-b014-f90606e3a723)

Reply:
idk about these, the orange one and the blue were better

Reply:
> ok, I would say that the blue one fits better. if y'all think otherwise tell me

I agree with you and if a few people don't its fairly trivial to change it in their config. Assuming that's alright with Alex.

Reply:
Make sure to use the `bold = true` and `fg = C.black0` options.

Reply:
![image](https://github.com/user-attachments/assets/36fedd54-20fc-4995-8800-888a00194dfa)
this is how it looks in my config

Reply:
i think it looks great

Reply:
That looks great to me! : )

Reply:
@AlexvZyl what do you think?

Reply:
![image](https://github.com/user-attachments/assets/8799deef-4921-4f3d-84cf-71651566072b)
@5-pebbles this is it right?

Reply:
> ![image](https://private-user-images.githubusercontent.com/70219992/371880236-8799deef-4921-4f3d-84cf-71651566072b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjc2MzM2NTcsIm5iZiI6MTcyNzYzMzM1NywicGF0aCI6Ii83MDIxOTk5Mi8zNzE4ODAyMzYtODc5OWRlZWYtNDkyMS00ZjNkLTg0Y2YtNzE2NTE1NjYwNzJiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTI5VDE4MDkxN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU4MmI5ZDk0ZDRiNGQxZDYyMjllM2NkODRjNjI1NjY3NjMzMjA2MjE5ZjIzY2FkNjhiNDJkZjNhMDc2MWY2MjkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.tA3EUWxov-h9zpJlT52aryEpgNgKwTBY06f7n-iH9z4) @5-pebbles this is it right?

I don't think the border should be bold.

Reply:
yeah that's true

Reply:
and what do y'all think about directories being blue?
![image](https://github.com/user-attachments/assets/1c0cc8c1-d0e8-464c-a083-529985613877)
just tried it and I think it doesn't look bad

Reply:
> and what do y'all think about directories being blue? ![image](https://private-user-images.githubusercontent.com/70219992/371881360-1c0cc8c1-d0e8-464c-a083-529985613877.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjc2MzQ1NDgsIm5iZiI6MTcyNzYzNDI0OCwicGF0aCI6Ii83MDIxOTk5Mi8zNzE4ODEzNjAtMWMwY2M4YzEtZDBlOC00NjRjLWEwODMtNTI5OTg1NjEzODc3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTI5VDE4MjQwOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI2ZGFlMThjNGEyNGJlZTBjZTY3N2Y5OTc3YmUwNDVjZGMyNzNiZDJhZDNkY2NmYzFlNDg0MGZmYTQwZmM0ODUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.gVOOESiyB-_3AyF9BJ5A3Q0QEx7Uu9o_1scQ2bBafWo) just tried it and I think it doesn't look bad

I don't think that's a good idea. Mostly just because it defies peoples expectations and is easy for them to set in their individual configs.

Reply:
yeah that's true, I thought about it and honestly makes it a little confusing for someone. and yeah, if someone wants it, he can use it for himself

Reply:
im just waiting for alex to confirm the new colors for titlebar and floatborder so i can push it

Reply:
Its like 9PM for him so that might be a while

Reply:
Me too lol